.circleci/config.yml

```diff
@@ -50,12 +51,46 @@ jobs:
                 cp -pR scenarioo-validator/build/test-results/test test-results/validator || true
                 cp -pR scenarioo-docu-generation-example/build/test-results/test test-results/docu || true
               when: always
+          - run:
+              name: Publish docu on demo.scenarioo.org/scenarioo-develop
+              command: |
+                pushd e2eScenariooDocu
+                zip -r ../e2eScenariooDocu.zip ./*
+                popd
+                curl -f --user scenarioo:$TOMCAT_USER_PASSWORD -F"file=@e2eScenariooDocu.zip" http://demo.scenarioo.org:8080/scenarioo-develop/rest/builds
+              when: always
           - store_test_results:
               path: test-results
           - store_artifacts:
               path: test-results
-          - store_artifacts:
-              path: e2eScenariooDocu
+              destination: /test-results
+
+          # Build Docker image with build WAR and generated docu
+          - setup_remote_docker:   # Creates a safe environment where docker commands are executed
+              docker_layer_caching: true # Speed up builds
+          # build and push Docker image
+          - run:
+              name: Create and push docker image
+              command: |
+                ENCODED_BRANCH=`echo $CIRCLE_BRANCH | tr / - | sed 's/#//g'`
+                TAG="$ENCODED_BRANCH-BUILD_$CIRCLE_BUILD_NUM"
+                DOCKER_BUILD_DIR="docker/scenarioo-ci"
+                DOCKER_HUB_REPO="cgrossde/scenarioo-dev"
+                echo "Copy WAR and scenarioo results of this build into docker image"
+                cp scenarioo-server/build/libs/scenarioo-latest.war $DOCKER_BUILD_DIR/scenarioo.war
+                # Keep the image smaller and only include E2E test results for now
+                # cp -pR scenarioo-docu-generation-example/build/scenarioDocuExample $DOCKER_BUILD_DIR
+                cp -pR e2eScenariooDocu/* $DOCKER_BUILD_DIR/scenarioDocuExample
+                cd $DOCKER_BUILD_DIR
+                echo "Build and push docker image to Docker Hub"
+                docker build -t $DOCKER_HUB_REPO:$TAG .
+                docker login -u $DOCKER_USER -p $DOCKER_PASS
+                docker push cgrossde/scenarioo-dev:$TAG
```

Comment:
I want this to go to our scenarioo account before we merge this
I do still not see the benefit of these scenarioo-dev dockers - wil we ever deploy that ???

Reply:
Instead of this I would prefare to have simple per branch snapshot artifacts of our real docker image - but without special docu data inside. Please remove this - unless we see a real benefit for this.

The docu is published to our demo, so I do not see the need for this.

Reply:
Didn't assign you on purpose to this PR, sorry. You can ignore this need/use this for testing atm, still experimenting ;) The need is simple: we want to run a demo and I'm still convinced that in the long run docker is easiest.

Reply:
BTW check this out: http://demo.scenarioo.org/scenarioo-develop/#/?branch=scenarioo-feature_755-investigate-building-docker-image&build=last%20successful&comparison=Disabled&tab=usecases

We can now deploy the E2E scenarioo date to `demo.scenarioo.org/scenarioo-develop/` from CircleCI

Reply:
you do not have to sorry - I think its good practice to have PRs early to get feedback early - so here you have my thoughts ;-)